core-test/src/test/scala/org/bitcoins/core/crypto/bip32/BIP32PathTest.scala

```diff
@@ -16,7 +16,7 @@ import scala.util.{Success, Try}
 class BIP32PathTest extends BitcoinSUnitTest {
 
   implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
-    generatorDrivenConfigNewCode
+    generatorDriveConfigOldCode
```

Comment:
Is this safe? Maybe a better idea might be to split `BIP32PathTest` into 2 separate files, one with ones we running 100 times, and ones we want running 20 times.